document floating windows (#7156)
microsoft/vscode-docs: docs/editor/custom-layout.md

![Editor Grid 2x2 layout with the sash highlighted](images/custom-layout/grid-editor-layout.png)

### Floating editor windows

> If you feel like moving an editor to another place on your monitor(s), you can just drag it out of the main window and drop it anywhere to open the editor in a floating window:

Reviewer: ```suggestion
You can open an editor in a floating window, for example to move the editor to another place on your monitor or even to another monitor.

To open an editor in a floating window, drag it out of the main window and drop it anywhere outside of the current VS Code window:
```

<video src="images/custom-layout/floating-windows.mp4" autoplay loop controls muted></video>

Floating editor windows are capable of opening as many editors as you like in a grid layout. The windows will restore at their location after restart and reopen all the editors within.

> Another way of detaching editors is by right-click into an editor tab and pick the option **Move into New Window** (`kb(workbench.action.moveEditorToNewWindow)`) or **Copy into New Window** (`kb(workbench.action.copyEditorToNewWindow)`).

Reviewer: ```suggestion
Another way to detach an editor is to right-click on an editor tab, and select the option **Move into New Window** (`kb(workbench.action.moveEditorToNewWindow)`) or **Copy into New Window** (`kb(workbench.action.copyEditorToNewWindow)`).
```

![Floating windows from editor tab menu](images/custom-layout/floating-windows.png)

> If you feel like moving an entire editor group, pick **Move Editor Group into New Window** (`kb(workbench.action.moveEditorGroupToNewWindow)`) or **Copy Editor Group into New Window** (`kb(workbench.action.copyEditorGroupToNewWindow)`).

Reviewer: ```suggestion
If you want to move an entire editor group, select **Move Editor Group into New Window** (`kb(workbench.action.moveEditorGroupToNewWindow)`) or **Copy Editor Group into New Window** (`kb(workbench.action.copyEditorGroupToNewWindow)`).
```
